optimization/generate_profile.py

```diff
@@ -105,38 +102,51 @@ def load_course_map(course_name="COTA"):
             pitch = numpy.arctan(float(clean_data[i][1]) / 10)
             elev_profile.append(pitch)
     if course_name == "ASC":
-       pass 
+        pass
     return elev_profile
  
 if __name__ == '__main__':
     elev_profile = load_course_map()
     # Load in the distance and necessary time for a lap
     distance = 5490
-    time = 420 # max allowable time in s
-    iterations = 500  # Number of iterations to use
+    time = 420
+    iterations = 500 # Number of iterations to use
```

Comment:
need to change iterations to whatever num it should be

Reply:
I'd leave in the time comment. As well, maybe make that a passed-in parameter

Reply:
Are time and max_race_time supposed to be different?

Reply:
Note here. They shouldn't be. Appears to be an artifact of testing. Should be fixed.